typed/merge.go

```diff
@@ -248,6 +239,46 @@ func (w *mergingWalker) visitListItems(t *schema.List, lhs, rhs value.List) (err
 	return errs
 }
 
+func (w *mergingWalker) visitListOperand(t *schema.List, list value.List) ([]fieldpath.PathElement, fieldpath.PathElementValueMap, ValidationErrors) {
```

Comment:
That doesn't mean much to me, I like the other name as much or maybe something like `indexListPathElements` or something?

Reply:
Sounds good to me!